src/util/search.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * SelectMultipleFilter needs to parse values from format
+ * "has_all:a,b,c,d" or "a,b,c,d"
+ */
+export const parseSelectFilterOptions = uriComponentValue => {
+  return uriComponentValue && uriComponentValue.indexOf(':') >= 0
+    ? uriComponentValue.split(':')[1].split(',')
+    : uriComponentValue && uriComponentValue.indexOf(':') < 0
+    ? uriComponentValue.split(',')
+    : [];
+};
```

Comment:
I think the above function fails if one of the values contains `:`. I don't know if that's possible, but I don't think anything restricts that, right?

To me the nested ternaries are really hard to follow. I get that avoiding naming things is good, but this is pushing way too much logic into a single expression IMHO.